test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py

```diff
@@ -72,7 +72,7 @@ def tfs_model(region, boto_session):
 def python_model_with_requirements(region, boto_session):
     return util.find_or_put_model_data(region,
                                        boto_session,
-                                       "data/python-with-requirements.tar.gz")
```

Comment:
this model supports tf26, right? you need add one more function `python_model_with_requirements_for_greater_than_equal_to_tf26` and use it in tf2.6 test.